### PR TITLE
Migration to Swift 3

### PR DESCRIPTION
**Pull request by nikolasburk**
This version works with latest Swift 3 syntax and Xcode 8.

### CODE REVIEW

File: ListKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -389,21 +389,24 @@
 			attributes = {
 				LastSwiftMigration = 0700;
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0700;
+				LastUpgradeCheck = 0800;
```
Comment (Ben-G) [inline ListKit.xcodeproj/project.pbxproj:392]: It seems like you're on Xcode 8, not 8.1?

If that's the case, would you mind to update this PR with Xcode 8.1 which has Swift 3.0.1 support? 😄

Comment (nikolasburk) [reply ListKit.xcodeproj/project.pbxproj:392]: ah sorry, I didn't realize that! will submit a new PR shortly, thanks a lot for the hint!

Comment (nikolasburk) [reply ListKit.xcodeproj/project.pbxproj:392]: I just upgraded my Xcode to `8.1` and rebuilt the project with that new version of Xcode. After the build, there are no changes in the git repository that I could commit though.

What exactly do I have to make sure the PR is coming from Xcode 8.1? I assume `LastUpgradeCheck` has to be set to `0810`?! Can this this be done manually or should it happen through an Xcode build? Are there other things to take care of?

**Comment by nikolasburk** [comment]
I believe I made the proper changes now. Will submit a new PR, closing this one.

**Pull request closed without merging**